tailwarden/komiser
# feat: allow user to manage existing cloud accounts from dashboard

## Problem

Currently users have no possibility to edit a cloud account

## Solution

[Provide a detailed explanation of the solution you are proposing, including any relevant code snippets or screenshots.]

## Changes Made



## How to Test

Please go to the `/cloud-accounts` page, click on the '...' icon to see actions, and select "Edit cloud account"

## Screenshots



https://github.com/tailwarden/komiser/assets/1955172/266c657c-5ef5-44d3-9a2b-bc9d140412e9




## Notes

[Any additional notes or information that you would like to share with the reviewers.]

## Checklist


## Reviewers

@[username of the reviewer]
